sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -94,6 +95,17 @@ internal class AmqpReceiver : TransportReceiver
         /// </summary>
         private readonly ConcurrentExpiringSet<Guid> _requestResponseLockedMessages;
 
+        private static IReadOnlyList<ServiceBusReceivedMessage> emptyList;
+
+        private static IReadOnlyList<ServiceBusReceivedMessage> NoMessagesReceived
```

Comment:
```suggestion
        private static IReadOnlyList<ServiceBusReceivedMessage> s_emptyList
```

Reply:
So the property lower case? Looks ugly :D

Reply:
Actually I think you are right - since this is a property rather than a field, PascalCase should be fine.